client/src/actions/actionsStore/TaskActions.js

```diff
@@ -38,3 +42,44 @@ export const createTask = (token, formProps, onSuccess) => async (dispatch) => {
     dispatch({ type: CREATE_TASK_FAILED, payload: "Could not create task" });
   }
 };
+
+// Update an existing task
+export const updateTask = (token, formProps, taskId, onSuccess) => async (
+  dispatch
+) => {
+  try {
+    const { title, description, startDate, dueDate } = formProps;
+    const data = {
+      title,
+      description,
+      startDate,
+      dueDate
```

Comment:
can we add a place for status, like maybe it's in progress, blocked, etc.

Reply:
@meganhng I think I might leave passing status in another PR, since this PR already changes a lot of things, and I'd like to avoid creating more bugs. Our current model for tasks doesn't contain a status property, so we can probably work on that after. It might also be better to decide how a task's status is set first, and how it is updated (Ex. Does a user change the status of a task manually, or does the task's status change if the user updates a task's start and due date). What do you think?